netsim/modules/bgp.py

```diff
@@ -221,7 +219,7 @@ def build_ebgp_sessions(node: Box, sessions: Box, topology: Box) -> None:
     node_as =  node.bgp.get("as")                                     # Get our real AS number and the AS number of the peering session
```

Comment:
The use of ```l``` (link) to iterate over ```node.interfaces``` always confuses me here. I would have picked ```i```